docs/advanced/input_files/input-main.md

```diff
@@ -1678,11 +1679,18 @@ These variables are used to control the output of properties.
 ### out_mat_xc
 
 - **Type**: Boolean
-- **Availability**: Numerical atomic orbital basis
+- **Availability**: Numerical atomic orbital (NAO) and NAO-in-PW basis
 - **Description**: Whether to print the upper triangular part of the exchange-correlation matrices in **Kohn-Sham orbital representation** (unit: Ry): $\braket{\psi_i|V_\text{xc}^\text{(semi-)local}+V_\text{exx}+V_\text{DFTU}|\psi_j}$ for each k point into files in the directory `OUT.${suffix}`, which is useful for the subsequent GW calculation. (Note that currently DeePKS term is not included. ) The files are named `k-$k-Vxc`, the meaning of `$k`corresponding to k point and spin  is same as [hs_matrix.md](../elec_properties/hs_matrix.md#out_mat_hs).
 The band (KS orbital) energy for each (k-point, spin, band) will be printed in the file `OUT.${suffix}/vxc_out`. If EXX is calculated, the local and EXX part of band energy will also be printed in `OUT.${suffix}/vxc_local_out`and `OUT.${suffix}/vxc_exx_out`, respectively. All the `vxc*_out` files contains 3 integers (nk, nspin, nband) followed by nk\*nspin\*nband lines of energy Hartree and eV.
```

Comment:
I would suggest adding ".dat" after these output file names

Reply:
The names "xxx_out" are consistent with "vxc_out" read by LibRPA. @minyez.

Reply:
It is completely fine for LibRPA to adapt the names of inputs or use symlink. If adding ".dat" makes it more consistent with other outputs in abacus, I would also suggest adding it.

Reply:
".dat" is added after all the "${term}_out" files in the newest commit.